doc/Makefile

```diff
@@ -42,6 +42,7 @@ help:
 	@echo "  old        create old-style HTML doc pages in old dir"
 	@echo "  fetch      fetch HTML and PDF files from LAMMPS web site"
 	@echo "  epub       create ePUB format manual for e-book readers"
+	@echo "  mobi       create MOBI format manual for e-book readers (e.g. Kindle)"
```

Comment:
maybe add "requires calibre for ebook-convert"

Reply:
done